script.js

```diff
@@ -1,4 +1,81 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+
+// generated SPS
+
+var generateRandomScissorsPaperStone = function(){
+var randomDecimal = Math.random() * 3
+var randomInteger = Math.floor (randomDecimal)
+var diceNumber = randomInteger +1 
+var RandomScissorsPaperStone = "undefined"
+
+if (diceNumber == 1){
+  RandomScissorsPaperStone = 'scissors' 
+};
+
+if (diceNumber == 2){
+  RandomScissorsPaperStone = 'paper'
+
+};
+if (diceNumber == 3){
+  RandomScissorsPaperStone = 'stone'
+ 
 };
+console.log (RandomScissorsPaperStone);
+return RandomScissorsPaperStone
+
+};
+
+
+/*
+
+// win or lose
+​
+var getResults = function(a, b){
```

Comment:
you can name this `var getResults = function (userPlays, computerPlays) {}` so that it's clearer what the parameters are. Good job with using two parameters in the function though!